servers/rendering/rasterizer_rd/rasterizer_canvas_rd.h

```diff
@@ -238,10 +238,10 @@ class RasterizerCanvasRD : public RasterizerCanvas {
 
 	struct TextureBindingKeyHasher {
 		static _FORCE_INLINE_ uint32_t hash(const TextureBindingKey &p_key) {
-			uint32_t hash = hash_djb2_one_64(p_key.texture.get_id());
-			hash = hash_djb2_one_64(p_key.normalmap.get_id(), hash);
-			hash = hash_djb2_one_64(p_key.specular.get_id(), hash);
-			hash = hash_djb2_one_64(p_key.multimesh.get_id(), hash);
+			uint32_t hash = uint32_t(hash_djb2_one_64(p_key.texture.get_id()));
+			hash = uint32_t(hash_djb2_one_64(p_key.normalmap.get_id(), hash));
+			hash = uint32_t(hash_djb2_one_64(p_key.specular.get_id(), hash));
+			hash = uint32_t(hash_djb2_one_64(p_key.multimesh.get_id(), hash));
```

Comment:
I don't think this is right. `hash_djb2_one_64` takes in a previous hash of type `uint64_t` and returns `uint64_t`, there's no reason to truncate to `uint32_t` on each step. I think a better solution would be to change `hash` to be `uint64_t` and then change the return line to be `return uint32_t(hash);` or similar.

(...or alternatively it's possible the method could return `uint64_t` as I asked about above)

Reply:
Seems that in code is a lot of places where uint32_t is used as hash due history reasons(32 bit computers), and I think that can be changed in Godot 4.0, but it will require some refactoring which goes beyond the scope of this PR.
![Zrzut ekranu z 2020-08-16 10-16-12](https://user-images.githubusercontent.com/41945903/90329967-97f36800-dfa9-11ea-9f04-8c53b10e8d5f.png)


https://github.com/godotengine/godot/blob/3f1fc5af7affe27bc2fbb1492377930cc33cc526/core/string_name.h#L100

https://github.com/godotengine/godot/blob/00949f0c5fcc6a4f8382a4a97d5591fd9ec380f8/core/callable.h#L116

https://github.com/godotengine/godot/blob/a6916d1f179f760f015f385c01729d205af97abe/core/variant.cpp#L2843